diff -ur app.src/AppKernel.php app.dst/AppKernel.php
--- app.src/AppKernel.php	2017-05-01 19:42:50.000000000 +0200
+++ app.dst/AppKernel.php	2017-05-13 23:05:36.229802779 +0200
@@ -15,7 +15,8 @@
             new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
             new Doctrine\Bundle\DoctrineBundle\DoctrineBundle(),
             new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
-            new AppBundle\AppBundle(),
+            new Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle(),
+            new Abienvenu\KyjoukanBundle\KyjoukanBundle(),
         );

         if (in_array($this->getEnvironment(), array('dev', 'test'), true)) {
@@ -23,6 +24,7 @@
             $bundles[] = new Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
             $bundles[] = new Sensio\Bundle\DistributionBundle\SensioDistributionBundle();
             $bundles[] = new Sensio\Bundle\GeneratorBundle\SensioGeneratorBundle();
+            $bundles[] = new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle();
         }

         return $bundles;
